src/Mod/Path/PathSimulator/App/VolSim.cpp

```diff
@@ -783,14 +783,14 @@ cSimTool::cSimTool(const TopoDS_Shape& toolShape, float res){
 
 float cSimTool::GetToolProfileAt(float pos)  // pos is -1..1 location along the radius of the tool (0 is center)
 {
-	try{
```

Comment:
Removing the try is probably OK, unless the comparison itself can raise exceptions. If it does the catch should also have reported the text of those.